metricbeat/docker-compose.yml

```diff
@@ -17,11 +17,11 @@ services:
 
   # Used by base tests
   elasticsearch:
-    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-7.14.0}-1
+    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-8.0.0}-1
```

Comment:
This does not seem in sync with the statement in the PR message which states 8.2

Reply:
I can't read, it seems, this should indeed be 8.2, @elastic/observablt-robots ?

Reply:
Actually I think its ok, because the 8.0.0 is released.